core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionalTable.java

```diff
@@ -99,7 +98,7 @@
  */
 public class TransactionalTable extends HTable implements TransactionalTableClient {
     static final Log LOG = LogFactory.getLog(RMInterface.class);
-    static private HConnection connection = null;
+    static private Connection connection = null;
     static Configuration       config = HBaseConfiguration.create();
     static ExecutorService     threadPool;
```

Comment:
Tried. But, It looks like there is no HTable constructor that takes byte[] and Connection.